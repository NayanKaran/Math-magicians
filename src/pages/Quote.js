import React from 'react';

const Quote = () => (
  <>
    <main>
      <p>
        Mathematics is not about numbers, equations, computations or
        algorithms; it&apos;s about understanding. - William Paul Thurston
      </p>
    </main>
  </>
);

export default Quote;
