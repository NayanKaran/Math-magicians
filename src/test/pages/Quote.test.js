import renderer from 'react-test-renderer';
import Quote from '../../pages/Quote';

describe('Testing Quote component', () => {
  it('Testing with snapshot', () => {
    const tree = renderer.create(<Quote />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
