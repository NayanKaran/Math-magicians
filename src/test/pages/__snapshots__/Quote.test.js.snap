// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Quote component Testing with snapshot 1`] = `
<main>
  <p>
    Mathematics is not about numbers, equations, computations or algorithms; it's about understanding. - William Paul Thurston
  </p>
</main>
`;
